app/src/main/java/com/pv239/beelocal/ui/screens/dailychallenge/DailyChallengeViewModel.kt

```
@@ -0,0 +1,244 @@
+                repository.submitDailyChallenge(completion, newStreak)
+
+                _uiState.update { state ->
+                    (state as? DailyChallengeUiState.Ready)?.copy(
+                        completion = CompletionState.Completed(
+                            photoUrl = photoUrl,
+                            streakCount = newStreak,
+                            sharedToFeed = false,
+                        )
+                    ) ?: state
+                }
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)